### PR TITLE
templates?

### PR DESCRIPTION
**Pull request by gintama91**

### CODE REVIEW

File: templates/class_template.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# class_template.erb
+# frozen_string_literal: true
+
+class Scarpe
```
Comment (noahgibbs) [inline templates/class_template.erb:4]: If we do templates, we probably also want display properties, plus binding an event.

**Comment by Schwad** [comment]
I approve of this idea and I really like it. I'll keep an eye on when it's marked ready for review, then I'll give it a spin locally.

Also in the templates I like the idea of adding commented sections to show other things that can be included maybe? Or even comments? but we can iterate. Would be cool to add something to the README development section as well.

+1 what noah said about including display properties, even if commented out

File: templates/class_template.erb (hunk 1 above)
Comment (gintama91) [reply templates/class_template.erb:4]: do we need binding an event for all? like for fill,background we dont ig?

Comment (gintama91) [reply templates/class_template.erb:4]: should i do like in comment?

**Comment by gintama91** [comment]
this is fun!!!!!

image

File: templates/class_template.erb (hunk 1 above)
Comment (noahgibbs) [reply templates/class_template.erb:4]: We definitely don't need bindings -- or maybe even display properties -- for literally every widget/object. In a comment would be fine for those, sure. It's a template, so it's good to include optional-but-common stuff too.

Comment (Schwad) [reply templates/class_template.erb:4]: Yeah I love the idea of having the options commented out. If you are familiar with Rails you can see examples of this in config/production.rb - though that's just one example of a nice and common idiom :)

**Comment by gintama91** [comment]
yayyy lol this is funnn, tried like with shoes app instead of terminal(inspired frm noah highlander)

image

**Comment by noahgibbs** [comment]
That's fun!

**Comment by gintama91** [comment]
image

**Comment by gintama91** [comment]
this is ready for review

**Comment by gintama91** [comment]
Actually on second thought i think this can be closed as all this code is in #244  (ig it it there)

closing in favour of #244

**Pull request closed without merging**